lib/dom.js

```diff
@@ -1919,12 +2067,26 @@ function _insertBefore(parent, node, child, _inDocumentAssertion) {
 	}
 	do {
 		newFirst.parentNode = parent;
+		if (parent.nodeType === parent.ELEMENT_NODE) {
+			newFirst.parentElement = parent;
+		} else {
+			newFirst.parentElement = null;
+		}
```

Comment:
I think it would be better to implement this as a getter, to avoid having to maintain this value when the DOM structure is changed.